cfme/containers/node.py

```diff
@@ -1,59 +1,124 @@
-        Returns: A string representing the contents of the InfoBlock's value.
+            *ident: Table name and Key name, e.g. "Relationships", "Images"
+        Returns: A string representing the contents of the summary's value.
         """
-        self.load_details(refresh=True)
-        return details_page.infoblock.text(*ident)
-
-    def navigate(self, detail=True):
-        if detail is True:
-            if not self._on_detail_page():
-                sel.force_navigate('containers_node_detail', context={'node': self})
-        else:
-            sel.force_navigate('containers_node', context={'node': self})
+        self.load_details(refresh=False)
+        return InfoBlock.text(*ident)
+
+
+@navigator.register(Node, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def am_i_here(self):
+        summary_match = summary_title() == '{} (Summary)'.format(self.obj.name)
+        return summary_match and match_title_and_controller()
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Compute', 'Containers', 'Container Nodes')(None)
+        Quadicon
+
+    def resetter(self):
+        # Reset view and selection
+        tb.select("List View")
+        sel.check(paginator.check_all())
+        sel.uncheck(paginator.check_all())
+
+
+@navigator.register(Node, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def am_i_here(self):
+        summary_match = summary_title() == '{} (Summary)'.format(self.obj.name)
+        return summary_match and match_title_and_controller()
+
+    def step(self):
+        list_tbl.click_row_by_cells({'Name': self.obj.name, 'Provider': self.obj.provider.name})
+
+
+@navigator.register(Node, 'EditTags')
+class EditTags(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Detail')
+
+    def am_i_here(self):
+        header_match = sel.is_displayed('//h3[text() == "Tag Assignment"]')
+        return header_match and match_title_and_controller()
+
+    def step(self):
+        pol_btn('Edit Tags')
+
+
+@navigator.register(Node, 'ManagePolicies')
+class ManagePolicies(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Detail')
+
+    def am_i_here(self):
+        header_match = sel.is_displayed('//h3[text() == "Select Policy Profiles"]')
+        return header_match and match_title_and_controller()
+
+    def step(self):
+        pol_btn('Manage Policies')
+
+
+@navigator.register(Node, 'Utilization')
+class Utilization(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Detail')
+
+    def am_i_here(self):
+        summary_match = summary_title() == '{} Capacity & Utilization'.format(self.obj.name)
+        return summary_match and match_title_and_controller()
+
+    def step(self):
+        mon_btn('Utilization')
+
+
+@navigator.register(Node, 'Timelines')
+class Timelines(CFMENavigateStep):
```

Comment:
Attempted this but the power up/down didn't seem to generate any events. will try again today.